pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go

```diff
@@ -171,6 +171,17 @@ type BackupJobs struct {
 	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
+
+	// Scheduling constraints of the Dedicated repo host pod.
+	// Changing this value causes repo host to restart.
```

Comment:
❓ These new fields are for jobs, rather than repo host, right?

Reply:
updated